README.md

```diff
@@ -240,25 +240,25 @@ should be used.
 | ------------------------------------------------------------------------ | ------------------------------ | ----------------------------------------------------- | --------------------------------------------------------------- | --------------------------------------------------------- | -------------------------------------------------------------------- | --------------------------------------------------- | ----------------------------------------------------- |
 | Include line numbers in log lines                                        | :+1:                           | :+1: (via user config)                                | :+1: (via user config)                                          | :+1:                                                      | :+1:                                                                 | :x:                                                 | :+1:                                                  |
 | Include other location information in log lines                          | :+1:                           | :x:                                                   | :+1: (via user config)                                          | :+1:                                                      | :+1:                                                                 | :x:                                                 | :+1:                                                  |
-| Persistent location counter between NeoVim sessions                      | :+1:                           | :x:                                                   | :x:                                                             | :x:                                                       | :x:                                                                  | :x:                                                 | :x:                                                   |
+| Persistent location counter between NeoVim sessions                      | :+1:                           | :x:                                                   | :x:                                                             | :x:                                                       | :+1: (depends on buffer state, not tracked internally in Lua)        | :x:                                                 | :x:                                                   |
```

Comment:
Thanks for this. What does this mean? As best as I can tell from the documentation, refactoring.nvim doesn't support a location counter at all (I mean a monotonically increasing counter to help distinguish lines). I can't actually get refactoring.nvim to work for me right now (I'm seeing `refactoring.nvim/lua/refactoring/debug/printf.lua:65: assertion failed!` whatever I do) so I can't see it working in practice...